bindings/swagger_parsing/load.py

```diff
@@ -0,0 +1,290 @@
+                    streaming = True
+
+                    # handle inlined objects similar to how swagger-codegen does
+                    result_type_class = classify_definition(
+                        enums, f"{name}.response.{code}", rschema
+                    )
+                    assert isinstance(result_type_class, Class)
+                    result_type_class_ref = result_type_class.params["result"].type
+                    assert isinstance(result_type_class_ref, Ref)
+                    pascal_name = result_type_class_ref.name
+                    pascal_name = pascal_name[0].upper() + pascal_name[1:]
+                    result_type_class.name = f"StreamResultOf{pascal_name}"
+                    result_type = Ref(result_type_class.name)
+                    extra_classes[result_type_class.name] = result_type_class
+
+                    responses[code] = result_type
+                    continue
+
+                responses[code] = classify_type(enums, f"{name}.responses.{code}", rschema)
+                if streaming is True:
+                    raise ValueError(
+                        f"a method must be either all-streaming or all-nonstreaming: {rspec}"
+                    )
+                streaming = False
+
+            if bad_op:
+                continue
+
+            assert streaming is not None
+
+            # Figure out parameters.
+            params = {}
+            for pspec in spec.get("parameters", []):
+                where = pspec["in"]
+                serialized_name = None
+                if where == "query":  # preserve query parameter names
+                    serialized_name = pname = pspec["name"]
+                pname = pspec["name"].replace(".", "_")
+                required = pspec.get("required", False)
+                if "schema" in pspec:
+                    pschema = pspec["schema"]
+                else:
+                    # swagger has some weird inlining going on here...
+                    inlined = ("type", "format", "items", "properties", "enum")
+                    pschema = {k: pspec[k] for k in inlined if k in pspec}
+                ptype = classify_type(enums, f"{name}.{pname}", pschema)
+                title = pspec.get("title") or pspec.get("description")
+                params[pname] = Parameter(pname, ptype, required, where, serialized_name, title)
+
+            assert is_expected_path(path), (path, name)
+            path = path.replace(".", "_")
+            tags = set(spec["tags"])
+            summary = spec["summary"]
+            needs_auth = "security" not in spec
+            op = Function(
+                name, method, path, params, responses, streaming, tags, summary, needs_auth
+            )
+            ops[name] = op
+    return ops, extra_classes
+
+
+def link_all_refs(defs: TypeDefs) -> None:
+    for ref in Ref.all_refs:
+        ref.linked = True
+        ref.defn = defs[ref.name]
+
+
+def load(path: str) -> ParseResult:
```

Comment:
recommend naming this "parse" instead of "load", since it returns a ParseResult.